LEAF_Request_Portal/api/controllers/IconPickerController.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/*
+ * As a work of the United States government, this project is in the public domain within the United States.
+ */
+
+require '../sources/IconPicker.php';
+
+if (!class_exists('XSSHelpers'))
+{
+    include_once dirname(__FILE__) . '/../../../libs/php-commons/XSSHelpers.php';
+}
+
+class IconPickerController extends RESTfulResponse
```

Comment:
Some of the coding standards for classes could apply here, not sure how return types would operate here at a glance. Maybe bring this up for discussion.